src/cljs/athens/effects.cljs

```diff
@@ -1,23 +1,28 @@
 (ns athens.effects
   (:require
-    [athens.config :as config]
-    [athens.datsync-utils :as dat-s]
-    [athens.db :as db]
-    [athens.util :as util]
-    [athens.walk :as walk]
-    [athens.ws-client :as ws]
-    [cljs-http.client :as http]
-    [cljs.core.async :refer [go <!]]
-    [cljs.pprint :refer [pprint]]
-    [clojure.string :as str]
+    [athens.common-events.schema :as schema]
+    [athens.config               :as config]
+    [athens.datsync-utils        :as dat-s]
+    [athens.db                   :as db]
+    [athens.self-hosted.client   :as client]
+    [athens.util                 :as util]
+    [athens.walk                 :as walk]
+    [athens.ws-client            :as ws]
+    [cljs-http.client            :as http]
+    [cljs.core.async             :refer [go <!]]
+    [cljs.pprint                 :refer [pprint]]
+    [clojure.string              :as str]
+    [com.stuartsierra.component  :as component]
     [dat.sync.client]
-    [datascript.core :as d]
-    [datascript.transit :as dt]
+    [datascript.core             :as d]
+    [datascript.transit          :as dt]
     [day8.re-frame.async-flow-fx]
-    [goog.dom.selection :refer [setCursorPosition]]
-    [posh.reagent :as p :refer [transact!]]
-    [re-frame.core :refer [dispatch reg-fx subscribe]]
-    [stylefy.core :as stylefy]))
+    [goog.dom.selection          :refer [setCursorPosition]]
+    [malli.core                  :as m]
+    [malli.error                 :as me]
+    [posh.reagent                :as p :refer [transact!]]
+    [re-frame.core               :as rf]
```

Comment:
I think we should refactor `[re-frame.core :refer [dispatch reg-fx subscribe]]` to `[re-frame.core :as rf]` in a separate PR here and in `events.cljs`.